### ISSUE TITLE
How to gradually appear filter during displacementFilter

### ISSUE THREAD

**Issue by michaelgrc** [pixijs/pixi.js#3262]
Hi everybody,

I've a logo in which I apply a displacement filter effect :
http://michaelg.fr/lab_webgl/displacementfilter/
On this example, I animated the displacementFilter during a onmouseover function.
When I'm not overing the logo, I would like to erase gradually the filter so the logo appears like in his initial looking (without curves). Indeed with my current code, my logo is always deformed.

Do you know which variable I've to modify to do this ?
In other words, do you know where the visual intensity of the filter is managed?

Thank you very much in advance for your help :)

**Comment by michaelgrc** [comment]
Up ?
Maybe @ivanpopelyshev could help me with his enigmas ? :)
Thanks very much, I tried to play with a lot of attributes but I'm king of stuck right now...

**Comment by michaelgrc** [comment]
Up again ? Sorry I can't find the solution my own...

**Comment by ivanpopelyshev** [comment]
sorry, was busy :) filter has its own "scale" field or something like that?

**Comment by michaelgrc** [comment]
There's absolutely no problems :)
Yeah filter has a scale property, I was thinking : maybe I can start with an elevate scale and then decrease it to obtain what I want.
But there must have a better way right ?

Finally I want my logo to be normal (not deformed) when it's not hovered.

Thank you very muchhhhh !

**Comment by GoodBoyDigital** [comment]
hello!

if you set the filter scale to 0 it should have no effect. Increasing this property will increase the strength of the filter.

Cheers!
